syntax = "proto3";
package NAlice;
option go_package = "a.yandex-team.ru/alice/megamind/protos/common";
option java_package = "ru.yandex.alice.megamind.protos.common";


// Yandex apps, initially which can be pushed with notification
enum EAppType {
    AT_UNDEFINED = 0;
    AT_SEARCH_APP = 1;
    AT_SMART_SPEAKER = 2;
    AT_MOBILE_BROWSER = 3;
    AT_DESKTOP_BROWSER = 4;
}
